README.md

```diff
@@ -113,75 +109,55 @@ Test artifacts deployment is handled by the set of shell scripts which are speci
 Lets consider **Scenario 02** from the Identity server scenarios [list](https://medium.facilelogin.com/thirty-solution-patterns-with-the-wso2-identity-server-16f9fd0c0389) as an example. Inside the [directory](https://github.com/wso2-incubator/identity-test-integration/tree/master/scenario02), we can see below hierarchy of files. All other scenarios should have the same hierarchy of contents.
 
 ```
-.
 ├── README.md
 ├── base-setup.sh
 ├── jmeter
-│   ├── 01-Scenario-02-Facebook.jmx
-│   ├── 02-Scenario-02-Google.jmx
-│   ├── 02-post-scenario-steps.sh
-│   └── 02-pre-scenario-steps.sh
+│   ├── 01-Scenario-02-Facebook.jmx
+│   ├── 02-Scenario-02-Google.jmx
+│   ├── 02-post-scenario-steps.sh
+│   ├── 02-pre-scenario-steps.sh
+│   ├── 03-Scenario-02-ISasIDP.jmx
+│   └── 04-Scenario-02-EmailOTP.jmx
 ├── resources
-│   └── user.properties
+│   └── user.properties
 └── teardown.sh
 
 ```
 
-Below is the correct order of the steps which TG runs the scripts of the scenarios.
+Below is the correct order of the steps which supposed to runs the scripts of the scenarios.
+
+**Step[1]**
+Check WSO2Scenariofile.yaml file and see whether the particular solution has associated config-set/s. If yes, then select relevat config set directory and copy it to the all(or relevant) product nodes andand run apply-config.sh. Note: PRODUCT_HOME should be provided as a parameter.
```

Comment:
if the scripts are running for the first time 'config-init.sh' should be executed to install the 'xmlstarlet' in the setup. Shall we include this in the readme?

Reply:
This is config02 and config03 specific and may not require for others. So mentioning it in the general guide may not require.

Reply:
According to the currently available config sets, config-init.sh is only applied for config2 and config3 as there is no change to the existing properties in config1. IMO if someone wants to execute the scenarios relevant to these config sets it would be better to provide some clue on the purpose of config-init.sh.